server/belga/io/feed_parsers/belga_tip_newsml_1_2.py

```diff
@@ -0,0 +1,166 @@
+import logging
+from .belga_newsml_1_2 import BelgaNewsMLOneFeedParser
+from superdesk.io.registry import register_feed_parser
+from superdesk.publish.formatters.newsml_g2_formatter import XML_LANG
+from .base_belga_newsml_1_2 import SkipItemException
+logger = logging.getLogger(__name__)
+
+
+class BelgaTipNewsMLOneFeedParser(BelgaNewsMLOneFeedParser):
+    """Feed Parser which can parse specific Belga News ML xml files."""
+
+    NAME = 'belgatipnewsml12'
+
+    label = 'Belga Tip News ML 1.2 Parser'
+
+    SUPPORTED_ASSET_TYPES = ('TIP')
```

Comment:
It must be a `('TIP',)` otherwise this statement: 
```
if not (role_name and role_name.upper() in self.SUPPORTED_ASSET_TYPES)
``` 
is equal to this 
```
if not (role_name and role_name.upper() in ['T', 'I', 'P']
```